carbonmark/lib/getPaymentMethods.ts

```diff
@@ -1,23 +1,36 @@
 import { CarbonmarkPaymentMethod } from "lib/types/carbonmark.types";
 import { StaticImageData } from "next/image";
+import BANK from "public/icons/Bank.png";
 import FIAT from "public/icons/Fiat.png";
 import USDC from "public/icons/USDC.png";
 
 export type CarbonmarkPaymentMethodMap = {
   [key in CarbonmarkPaymentMethod]: {
     id: string;
     icon: StaticImageData;
-    label: Uppercase<CarbonmarkPaymentMethod> | "Credit Card";
+    label: Uppercase<CarbonmarkPaymentMethod> | "Credit Card" | "Bank Transfer";
     disabled: boolean;
   };
 };
 
 export const carbonmarkPaymentMethodMap: CarbonmarkPaymentMethodMap = {
   usdc: { id: "usdc", icon: USDC, label: "USDC", disabled: false },
```

Comment:
Why not put the minimum amounts here? This seems clean.

Reply:
@biwano yeah this actually seems like a great idea to keep this logic together here. I'm going to create a separate ticket to cleanup the validation logic and move it out to this file, would prefer not to start making changes to this logic now as I believe we are looking to release this feature in todays release if possible.